### PR TITLE
ONAV-1040: Split API Endpoints into separate child pages

### PR DESCRIPTION
**Pull request by jhiggins-cpr**

### CODE REVIEW

**Review by civerachb-cpr — APPROVED** [review]
Looks OK to me.

File: docs_outdoornav_user_manual/api/api_endpoints/api_endpoints.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+toc_max_heading_level: 4
+---
+
+The ROS1 Noetic API can be used to monitor and manage OutdoorNav Software. Details
+are available through the child pages.
+
+- [Platform API](./platform_api)
+- [Autonomy API](./autonomy_api)
+- [Mission Manager API](./mission_manager_api)
+- [Definitions](./definitions)
```
Comment (jmastrangelo-cpr) [inline docs_outdoornav_user_manual/api/api_endpoints/api_endpoints.mdx:15]: all of these links were not working for me in my local build

**Review by jmastrangelo-cpr — COMMENTED** [review]
1) In the table of contents, the API Overview page shold be before the API Endpoints page.

2) The links in the API Overview page to Platform/autonomy/mission manager should send you to the relevant new page not the API Endpoints page.

**Comment by jhiggins-cpr** [comment]
> 1. In the table of contents, the API Overview page shold be before the API Endpoints page.
> 2. The links in the API Overview page to Platform/autonomy/mission manager should send you to the relevant new page not the API Endpoints page.

Fixed these two issues

File: docs_outdoornav_user_manual/api/api_endpoints/api_endpoints.mdx (hunk 1 above)
Comment (jhiggins-cpr) [reply docs_outdoornav_user_manual/api/api_endpoints/api_endpoints.mdx:15]: I'll follow up in Slack

**Review by jmastrangelo-cpr — APPROVED** [review]

File: docs_outdoornav_user_manual/api/api_endpoints/api_endpoints.mdx (hunk 1 above)
Comment (jhiggins-cpr) [reply docs_outdoornav_user_manual/api/api_endpoints/api_endpoints.mdx:15]: Confirmed with @jmastrangelo-cpr that this appears to be working ok now. I'll validate on the automated build after merging.

**Pull request merged**